Special abilities PR 5, Split Speed values (#8)
GameBrains/er-core: modules/special-rules/pages/save.adoc

= Save

These units can beat the odds and escape destruction -- perhaps through exceptional armour, protective fields, or extreme skills and toughness.

---

Where your unit has the {save} ability:

* Each time the unit suffers a hit, from any source, roll a die.
On a roll of 4+, ignore the hit -- it causes no damage.

> 
> NOTE: Attacks which ignore void shields or power fields do *not* ignore the {save} ability.

Author: Just spotted that there is already a bullet for this in holo-fields which we can borrow:

```suggestion
* The {save} ability applies to attacks that ignore void shields and power fields.
```

Author: This change looks fine, but Holo-fields now uses a completely different format from Save for a nearly-identical ability.

Reviewer: Looks like the exact same format to me -- it's just that Holo-fields has more clauses because it is more complex what with the automatic critical damage and all, and thus not a 'nearly-identical' ability.

In terms of process, I do agree that we want the presentation of similar things to be consistent — but the first priorities should be for each topic to be accurate and clear. So, it can be that when you're dipping in an out as you find time, and as you encounter new clauses and issues for which previous phrasing doesn't work well, it becomes inconsistent in these respects.

It's good to catch this as we go along where we can, but we shouldn't be **slaves** to it. That is, we can fix it in on a later iteration once we're able to step back and see the whole section -- this can be more efficient than constantly going go back and forth to tinker with it while fundamentals are still in flux during our first go at it.

So, good to note where we have inconsistencies, I agree -- but we shouldn't necessarily jump to fix them across the branch immediately every time.

Author: > Looks like the exact same format to me -- it's just that Holo-fields has more clauses because it is more complex what with the automatic critical damage and all, and thus not a 'nearly-identical' ability.

I think this was a case of crossed-wires. My comment about different formats was about the ability to ignore hits on a roll of X+, not about the note! See the Holo-fields conversation for more detail, and issues about Github not showing which lines I was referring to. :-(

> It's good to catch this as we go along where we can, but we shouldn't be **slaves** to it. That is, we can fix it in on a later iteration once we're able to step back and see the whole section -- this can be more efficient than constantly going go back and forth to tinker with it while fundamentals are still in flux during our first go at it.

Sure. It's just that, for me, Save and Holo-fields are variants of the same rule so I automatically think about both of them at the same time.

Reviewer: >for me, Save and Holo-fields are variants of the same rule so I automatically think about both of them at the same time.
Yeah, that's fair enough, and logical. Ideally I look at 'similar' topics in parallel, and often do -- but sometimes I'm in a hurry or I forget because of where my focus is at that particular moment.
